xcape/core.py

```diff
@@ -75,14 +76,16 @@ def _cape_dummy(p, t, td, ps, ts, tds, source, ml_depth, adiabat, pinc, type_gri
     return cape, cin, mulev, zmulev
 
 def calc_cape(p, t, td, ps, ts, tds, source='surface', ml_depth=500., adiabat='pseudo-liquid',
-         pinc=1000., method='fortran', vertical_lev='sigma'):
+         pinc=1000., method='fortran', vertical_lev='sigma', pres_lev_pos=1):
     """
```

Comment:
pres_lev_pos is set = 1 because for the vertical_lev='sigma' case (-> type_grid=1) won't be used anyways.